frontend/src/app/shared/components/utm/config/app-config-sections/app-config-sections.component.html

```diff
@@ -21,12 +21,15 @@ <h6 class="font-weight-semibold mb-2">{{section.section | titlecase}}</h6>
         Delete config
       </button>
 
-      <app-utm-email-conf-check [validConfig]="checkConfigValid()"
+      <app-utm-email-conf-check [validConfig]="(!checkConfigValid() || saving || configToSave.length === 0) && isCheckedEmailConfig"
+                                [configToSave]="configToSave"
+                                [emailConfig]="configs"
+                                (isChecked) = "checkedEmailConfig($event)"
                                 class="mr-2"
                                 *ngIf="section.section.toLowerCase() === 'email'"></app-utm-email-conf-check>
 
       <button (click)="saveConfig()"
-              [disabled]="!checkConfigValid() ||saving || configToSave.length === 0"
+              [disabled]="!checkConfigValid() || saving || configToSave.length === 0 || (section.section.toLowerCase() === 'email' && !isCheckedEmailConfig)"
```

Comment:
does the section have short name?? if true, use the short name

Reply:
No, the section does not have a short name.